pkg/bundle/image.go

```diff
@@ -0,0 +1,19 @@
+package bundle
+
+import (
+	"strings"
+
+	"github.com/openshift/library-go/pkg/image/reference"
+)
+
+// IsBlocked will return a boolean value on whether an image
+// is specified as blocked in the BundleSpec
+func IsBlocked(cfg *BundleSpec, imgRef reference.DockerImageReference) bool {
+
+	for _, block := range cfg.BlockedImages {
+		if strings.Contains(imgRef.Name, block) {
```

Comment:
Probably better to directly compare names rather than fuzzy matching, since one character set can be a strict subset of another and have this evaluate to true when, which I don't think is desirable behavior right? Ex `strings.Contains("specific-image-name", "reg/ns/my-specific-image-name:latest") == true` is bad because the images are different.
```suggestion
		if imgRef.Name.Exact() == block {
```

Reply:
@estroz I agree here. I was going back and forth on this one because if the blocked name is something generic or small if might get a false positive. I will add this one.